doc/source/user_guide.rst

```diff
@@ -3,30 +3,25 @@
 User guide
 ==========
 
-The ``pygranta`` metapackage itself has no functionalities. It is only a bundle of the
-different public PyGranta packages that are compatible with a given Granta MI release, depending
-on the version requested.
+The ``pygranta`` metapackage itself is not used directly. Instead, it provides a bundle of
+PyGranta packages that are compatible with each other and with a given Granta MI release.
 
-The documentation for each PyGranta package provides usage information. You can click
-the following cards to access their specific documentation.
+The user guide for each PyGranta package provides general usage information.
 
-**********************
-PyGranta documentation
-**********************
+********************
+PyGranta user guides
+********************
 
 .. grid:: 3
 
-    .. grid-item-card:: PyGranta BoM Analytics: documentation
-      :link: https://bomanalytics.grantami.docs.pyansys.com/version/stable/index.html
-      :text-align: center
+    .. grid-item-card:: PyGranta BoM Analytics :material-regular:`science` :material-regular:`eco`
+      :link: https://bomanalytics.grantami.docs.pyansys.com/version/stable/user_guide/index.html
```

Comment:
These links don't exist yet for all packages, but they will once this is released.